### PR TITLE
[CIS GCP] Notify the user about cloud shell editor deprecation

### PR DESCRIPTION
**Pull request by uri-weisman**
Update the GCP onboarding docs to notify our users about the GCP Cloudshell editor deprecation.
This change may confuse users who will install our integration using the deployment manager.

Slack discussion: https://elastic.slack.com/archives/C03E5KGNWT1/p1708447923318629

### CODE REVIEW

**Comment by uri-weisman** [comment]
cc @smriti0321 @benironside @tehilashn

File: docs/cloud-native-security/cspm-get-started-gcp.asciidoc [hunk 1]
```diff
@@ -59,8 +59,15 @@ For most users, the simplest option is to use a Google Cloud Shell script to aut
 +
 image::images/cspm-cloudshell-trust.png[The cloud shell confirmation popup]
 +
+NOTE: Google has deprecated its old Cloud Shell editor. If you continue to use it, you may encounter the following pop-up:
++
+image::images/cspm-cloudshell-old-editor.png[The cloud shell switch editor popup]
++
+If the popup appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started-gcp.asciidoc:66]: Inconsistency between `pop-up` and `popup`, but we could just call it a "message" instead. The fact that it pops up isn't super important.
```suggestion
NOTE: Google has deprecated its old Cloud Shell editor. If you continue to use it, you may encounter the following message:
+
image::images/cspm-cloudshell-old-editor.png[The cloud shell switch editor popup]
+
If the message appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```

File: docs/cloud-native-security/cspm-get-started-gcp.asciidoc [hunk 2]
```diff
@@ -59,8 +59,15 @@ For most users, the simplest option is to use a Google Cloud Shell script to aut
...
 . In Google Cloud Shell, execute the command you copied. Once it finishes, return to {kib} and wait for the confirmation of data received from your new integration. Then you can click **View Assets** to see your data.
 
+NOTE: If you encounter any issues running the command, return to Kibana and navigate again to Google Cloud Shell.
```
Comment (joepeeples) [inline docs/cloud-native-security/cspm-get-started-gcp.asciidoc:69]:
```suggestion
NOTE: If you encounter any issues running the command, return to {kib} and navigate again to Google Cloud Shell.
```

File: docs/cloud-native-security/cspm-get-started-gcp.asciidoc (hunk 1 above)
Comment (nastasha-solomon) [inline docs/cloud-native-security/cspm-get-started-gcp.asciidoc:66]:
```suggestion
If the popup appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new Editor, your context should remain unchanged.
```

**Review by nastasha-solomon — APPROVED** [review]
Just have one more to add to Joe's edit.

File: docs/cloud-native-security/cspm-get-started-gcp.asciidoc [hunk 3]
```diff
@@ -59,8 +59,15 @@ For most users, the simplest option is to use a Google Cloud Shell script to aut
 +
 image::images/cspm-cloudshell-trust.png[The cloud shell confirmation popup]
 +
+NOTE: Google has deprecated its old Cloud Shell editor. If you continue to use it, you may encounter the following message:
++
+image::images/cspm-cloudshell-old-editor.png[The cloud shell switch editor popup]
++
+If the message appears, click **X** or **Try the new editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```
Comment (benironside) [inline docs/cloud-native-security/cspm-get-started-gcp.asciidoc:66]:
```suggestion
If the message appears, click **X** or **Try the new Editor** and follow the next steps. When you switch to the new editor, your context should remain unchanged.
```

**Review by benironside — APPROVED** [review]

**Review by benironside — APPROVED** [review]

**Review by joepeeples — APPROVED** [review]

**Review by jmikell821 — APPROVED** [review]
Merged feedback LGTM!

**Pull request merged**